src/util/strencodings.cpp

```diff
@@ -565,3 +565,31 @@ std::optional<uint64_t> ParseByteUnits(const std::string& str, ByteUnit default_
     }
     return *parsed_num * unit_amount;
 }
+
+int64_t LocaleIndependentAtoi64(const std::string& str)
+{
+    int64_t result;
+    // Emulate atoi(...) handling of white space and leading +/-.
+    std::string s = TrimString(str);
+    if (!s.empty() && s[0] == '+') {
+        if (s.length() >= 2 && s[1] == '-') {
+            return 0;
+        }
+        s = s.substr(1);
+    }
+
+    auto [_, error_condition] = std::from_chars(s.data(), s.data() + s.size(), result);
```

Comment:
std::from_chars uses strtol locale dependent behavior, no?

Reply:
https://en.cppreference.com/w/cpp/utility/from_chars says

> 1) Integer parsers: Expects the pattern identical to the one used by std::strtol in the default ("C") locale and the given non-zero numeric base, except that

Reply:
It always uses the "C" locale, so it is locale-independent.

Reply:
gotcha.